src/autolabel/tasks/classification.py

```diff
@@ -55,6 +60,40 @@ def construct_prompt(self, input: Dict, examples: List) -> str:
         # prepare task guideline
         labels_list = self.config.labels_list()
         num_labels = len(labels_list)
+
+        # if large number of labels, filter labels_list by similarity of labels to input
+        if num_labels >= 50:
```

Comment:
can we do this based on some config setting. just want to make sure that we have the ability to turn this on or off.  we can do this on num_labels > 50 if the config setting corresponding to this is not set

Reply:
yes, we should enable this "label selection" from a config parameter, not a hardcoded num_labels threshold

Reply:
done in [f97a82d](https://github.com/refuel-ai/autolabel/pull/561/commits/f97a82d57b03bc3da5c59604b23ed7a573d230f7)

can now turn on/off label selection in config (as well as the number of labels to select), like so:
```
        "label_selection": true,
        "label_selection_count": 10
```